tests/unit-tests/src/test/java/org/apache/activemq/artemis/tests/unit/ra/ActiveMQResourceAdapterConfigTest.java

```diff
@@ -457,10 +456,7 @@ public void testConfiguration() throws Exception {
             newConfig.append("         \"         <config-property-value></config-property-value>\" + \n");
             newConfig.append("         \"      </config-property>\" + \n");
          }
-         System.out.println(newConfig);
          fail("methods not shown please see previous and add");
```

Comment:
this is only if failed, right? can't we keep it? it will tell us what is the exactly config that needs to be added (when only there's a failure)